addons/binding/org.openhab.binding.network/README.md

```diff
@@ -1,7 +1,7 @@
 # Network Binding
 
 This binding allows to check, whether a device is currently available on the network.
-This happens by either using [ping](https://en.wikipedia.org/wiki/Ping_%28networking_utility%29) or by a successful TCP connection on a port of your choosing.
+This happens by either using [ping](https://en.wikipedia.org/wiki/Ping_%28networking_utility%29), [arping](https://en.wikipedia.org/wiki/Arping) or by a successful TCP connection on a port of your choosing.
```

Comment:
Yes that's better. Maybe it would be better to add a new paragraph or headline somewhere around [here](https://github.com/Pakast/openhab2-addons/blame/a9b2b19726293b70a608cd0802f2f08df1c3bd7f/addons/binding/org.openhab.binding.network/README.md#L63) describing the difference between the three methods.